backend/test/logger.test.js

```diff
@@ -66,26 +67,45 @@ describe('logger', () => {
         const arg2 = Error();
         arg2.stack = 'secondArgStack';
         logger.info(arg1, arg2);
-        // eslint-disable-next-line no-console
-        expect(console.log).toHaveBeenCalledWith(expect.stringMatching(`${arg1} ${arg2.stack}`));
+        expect(console.log).toHaveBeenCalledWith(expect.stringContaining(`${arg1} ${arg2.stack}`));
```

Comment:
`stringContaining` is probably more accurate to `stringMatching`, as `stringMatching` argument is always converted into regular expression, so in case some regexp meaningful characters are included in the argument we can get falsy results.

Probably even better would be to create a helper function to first cut prefix (timestamp + log category) from the `console.log` and then strictly verify the rest of the text.